Source/Core/VideoBackends/Vulkan/VulkanContext.cpp

```diff
@@ -426,6 +432,7 @@ bool VulkanContext::SelectDeviceExtensions(ExtensionList* extension_list, bool e
   if (enable_surface && !CheckForExtension(VK_KHR_SWAPCHAIN_EXTENSION_NAME, true))
     return false;
 
+  CheckForExtension(VK_NV_GLSL_SHADER_EXTENSION_NAME, false, &m_supports_nv_glsl_extension);
```

Comment:
Why the change to the function when this would work just as well?

```
m_supports_nv_glsl_extension = CheckForExtension(VK_NV_GLSL_SHADER_EXTENSION_NAME, false);
```